LEGALNOTICE.md

```diff
@@ -0,0 +1,69 @@
+COPYRIGHT
+---------
+
+OWASP Zed Attack Proxy (ZAP)
+
+The software package is:
+
+    Copyright (C) 2010  Simon Bennetts
```

Comment:
Oh yeah - definitely need that!
Can you rip off some of the statements used here: https://github.com/zaproxy/zap-core-help/wiki/HelpParos ?